data/summary/types/cgm_test.go

```diff
@@ -94,6 +96,36 @@ func NewDataSetCGMDataRanges(startTime time.Time, hours float64, ranges DataRang
 	return dataSetData
 }
 
+func NewDataSetCGMVariance(startTime time.Time, hours int, perHour int, SD float64) ([]data.Datum, float64) {
+	requiredRecords := hours * perHour
+	typ := pointer.FromString("cbg")
+
+	var dataSetData = make([]data.Datum, requiredRecords)
+	var uploadId = test.RandomSetID()
+	var deviceId = "SummaryTestDevice"
+
+	// dedicated rand, to be sure the sequence is kept just in case.
+	r := rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
nit: Recording/printing this seed in the event of a failure could be useful... But I guess it could always be done later too, when there's an error that's found.

Reply:
I was expecting when this function was originally coded that it would generate much closer to requested StDev, but it results in quite far off requested values even with its own rand, it would require a completely different approach to generate a specific deviation, so this should just use the default rand, which is already printed at the start of the test.

I've removed this line.